client/src/components/BylineMetaBox/BylineProfiles.js

```diff
@@ -131,92 +137,102 @@ class BylineProfiles extends Component {
 
     return (
       <div>
-        <div className="byline-list-controls">
-          <div className="profile-controls">
+        <div className="byline-list-controls components-base-control">
+          <div className="profile-controls components-base-control__field">
             {/* eslint-disable jsx-a11y/label-has-for */}
-            <label htmlFor="profiles_autocomplete">
+            <label
+              className="components-base-control__label"
+              htmlFor="profiles_autocomplete"
+            >
               {window.bylineData.addAuthorLabel}
-              <Autocomplete
-                inputProps={inputProps}
-                items={this.state.searchResults}
-                value={this.state.search}
-                getItemValue={(item) => item.name}
-                wrapperStyle={{ position: 'relative', display: 'block' }}
-                onSelect={(value, item) => {
+            </label>
+            <Autocomplete
+              inputProps={inputProps}
+              items={this.state.searchResults}
+              value={this.state.search}
+              getItemValue={(item) => item.name}
+              wrapperStyle={{ position: 'relative', display: 'block' }}
+              onSelect={(value, item) => {
+                this.setState((state) => ({
+                  search: '',
+                  searchResults: [],
+                  profiles: [
+                    ...state.profiles,
+                    item,
+                  ],
+                }));
+              }}
+              onChange={(event, value) => {
+                clearTimeout(this.delay);
+                this.setState({
+                  search: value,
+                });
+
+                this.delay = setTimeout(() => {
+                  this.doProfileSearch(value);
+                }, 500);
```

Comment:
Possible to use a hook like `useDebounce` here instead?

Reply:
Perhaps, I'll look into it.